docs/install/user_auth_system.md

```diff
@@ -80,6 +80,18 @@ OPENID_CALLBACK_URL=/oauth/openid/callback
 8. Put the Client ID and Client Secret in the .env file
 9. Save the .env file
 
+---
+
+## How to Set Up Discord Authentication
+
+1. Go to [Discord Developer Portal](https://discord.com/developers)
+2. Create a new Application and give it a name
+4. In the OAuth2 general settings add a redirect URL and set it as "[Your DOMAIN_CLIENT](https://github.com/danny-avila/LibreChat/blob/main/.env.example#L219)/oauth/discord/callback" (example: http://localhost:3080/oauth/discord/callback)
```

Comment:
as @fuegovic mentioned on discord, .env.example should be updated, too. What i said above is in addition to that, as it would be helpful to have them in the instructions